config/locales/sv.yml

```diff
@@ -477,6 +477,15 @@ sv:
       mindfulness: mindfulness
       journaling: journalföring
       mood_tracking: humörspårning
+      veterans: veteraner
+      music: musik
+      integrative: integrativ
+      ptsd: PTSD
+      tbi: TBI
+      news: nyheter
+      government: regering
+      christian: kristen
+      humor: humör
```

Comment:
This one is a tricky one:

`humor: humör` => `humor: humor`

(humör == mood)

Reply:
yeah I know... I was thinking about this too. What is funny humor?

Reply:
The comedy kind of humor is called "humor".

With dots – "humör" – it's something else; mood.